BATTLES.md

```diff
@@ -0,0 +1,66 @@
+# Battles
+
+A guide to battles.
```

Comment:
I was hoping you'd elaborate a little bit on how the battles are implemented. Despite being a guide on battles, this doesn't actually mention `BattleMovr` at all. That's useful info.